File: April_2022/17_04_2022/Arrange_the_tiles.cpp
#include <iostream>

using namespace std;

int find(int arr[],int find,int n){
    for(int i=0;i<n;i++){
        if(arr[i] == find){
            return i;
        }
    }
}

int main(){

    int n,count=0;
    cin>>n;

    int current[n],desired[n];

    for(int i=0;i<n;i++){
        cin>>current[i];
    }

    for(int i=0;i<n;i++){
        cin>>desired[i];
    }

    for(int i=0;i<n;i++){
        if(current[i] == desired[i]){
            continue;
        }
        else{
            int x = find(current,desired[i],n);
            swap(current[i],current[x]);
            count++;
        }
    }
    
    cout<<count;

    return 0;
}

File: April_2022/17_04_2022/Arrange_the_tiles.py
n = int(input())
current = list(map(int,input().split()))
desired = list(map(int,input().split()))
count = 0
for i in range(n):
    if(current[i] == desired[i]):
        continue
    else:
        x = current.index(desired[i])
        current[i],current[x] = current[x],current[i]
        count += 1
print(count)